docs/content/https-tls/acme.md

```diff
@@ -169,9 +169,10 @@ Do not hesitate to complete it.
 | [VULTR](https://www.vultr.com)                              | `vultr`        | `VULTR_API_KEY`                                                                                                                             | Not tested yet                 |
 | [Zone.ee](https://www.zone.ee)                              | `zoneee`       | `ZONEEE_API_USER`, `ZONEEE_API_KEY`                                                                                                         | YES                            |
 
-[^1]: more information about the HTTP message format can be found [here](https://go-acme.github.io/lego/dns/httpreq/)
-[^2]: [providing_credentials_to_your_application](https://cloud.google.com/docs/authentication/production#providing_credentials_to_your_application)
-[^3]: [google/default.go](https://github.com/golang/oauth2/blob/36a7019397c4c86cf59eeab3bc0d188bac444277/google/default.go#L61-L76)
+[^1]: more information about the HTTP message format can be found [here](https://go-acme.github.io/lego/dns/httpreq/)  
+[^2]: [providing_credentials_to_your_application](https://cloud.google.com/docs/authentication/production#providing_credentials_to_your_application)  
+[^3]: [google/default.go](https://github.com/golang/oauth2/blob/36a7019397c4c86cf59eeab3bc0d188bac444277/google/default.go#L61-L76)  
+[^4]: as there is no way to support terminal attached to container when deploying with `docker stack`, you might resort to first running the container interactively (that is with `docker run -it`) to generate certificates and then reuse the certificates inside your docker stack container. You can mount a named volume with the same name containing the acme storage file to both containers to achieve this.
```

Comment:
```suggestion
[^4]: when deploying with `docker stack`, as there is no way to support terminal attached to container.
```

Reply:
I would leave the `docker run -it` part in the footnote to avoid confusion or put something equally clarifying. Firstly docker is already mentioned here. Secondly, the whole point is that "interactively" word IMO is to fuzzy to map to `docker run -it ...`  intuitively, and there are other commands that might appear "interactive" like `docker attach`  or `docker start -ai ...`, but they wouldn't work. I basically was going round in circles before @dduportal hinted that just attaching system streams might not be quite the same as doing `docker run -it ...`. After all, the fact is that LEGO is in fact launched "interactively" inside the container anyway, it's hard to point out when it's "interactive" enough, on the other hand it's very easy to figure out when it's `docker run -it` enough ;)

Reply:
you can add the `docker run -it` in the footnote